src/main/java/edu/greenblitz/robotName/subsystems/shooter/FlyWheel/FalconFlyWheel/FalconFlyWheelConstants.java

```diff
@@ -12,70 +9,85 @@ public static class rightMotorConstants{
         public static final int ID = 0;
         public static final NeutralModeValue NEUTRAL_MODE_VALUE = NeutralModeValue.Coast;
 
-        public static final MotionMagicConfigs MOTION_MAGIC_CONFIGS = new MotionMagicConfigs();
+        public static final Slot0Configs SLOT_0_CONFIGS = new Slot0Configs();
         static {
-            MOTION_MAGIC_CONFIGS.MotionMagicAcceleration = 1;
-            MOTION_MAGIC_CONFIGS.MotionMagicCruiseVelocity = 2;
-            MOTION_MAGIC_CONFIGS.MotionMagicJerk = 4;
+            SLOT_0_CONFIGS.kS = 1;
+            SLOT_0_CONFIGS.kV = 1;
+            SLOT_0_CONFIGS.kA = 1;
+            SLOT_0_CONFIGS.kP = 1;
+            SLOT_0_CONFIGS.kI = 0;
+            SLOT_0_CONFIGS.kD = 0;
         }
-
-        public static final ClosedLoopRampsConfigs CLOSED_LOOP_RAMPS_CONFIGS = new ClosedLoopRampsConfigs();
+        public static final MotionMagicConfigs MOTION_MAGIC_CONFIGS = new MotionMagicConfigs();
         static {
-            CLOSED_LOOP_RAMPS_CONFIGS.DutyCycleClosedLoopRampPeriod = 1;
-            CLOSED_LOOP_RAMPS_CONFIGS.TorqueClosedLoopRampPeriod = 2;
-            CLOSED_LOOP_RAMPS_CONFIGS.VoltageClosedLoopRampPeriod = 3;
+            MOTION_MAGIC_CONFIGS.MotionMagicAcceleration = SLOT_0_CONFIGS.kA;
+            MOTION_MAGIC_CONFIGS.MotionMagicCruiseVelocity = SLOT_0_CONFIGS.kS;
```

Comment:
I dont think we need cruise velocity, and if we doo we need it as max RPM,  no?

Reply:
yes, its the max RPM